CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added ACME certificate renewals and ACME account registration using external account binding
 - Added functionality to automatically renew ACME certificates.
 - Added an endpoint for statuses on asynchronous jobs and applied it to the ACME renewal endpoint.
+- Added DNSSEC support to CDN in a Box and enabled it
```

Comment:
"Support" is a strong word. CiaB doesn't truly "support" DNSSEC, since CiaB does its own DNS and doesn't tie in to real DNS zones.

Reply:
Agreed, weakened wording in 3e9988e473.